views/xTree/XTreeView.js

```diff
@@ -407,6 +407,8 @@ window.XTreeAncestorList = class XTreeAncestorList {
         // If the profile is private and the viewing user is not on the Trusted List, we still might not be able to continue.
         let p = data[0].person;
         XTreeView.PrimaryPerson = p;
+        console.log(p);
```

Comment:
Do you really want to log every person to the console in production?

Reply:
Oops - left in by mistake.  Commented out now.

Reply:
Fixed